src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -708,6 +708,32 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 eventQueue:=eventQueue)
         End Function
 
+        Friend Overrides Sub SerializePdbEmbeddedCompilationOptions(builder As BlobBuilder)
+            WriteValue(builder, "checked", Options.CheckOverflow.ToString())
+            WriteValue(builder, "langversion", LanguageVersion.ToDisplayString())
+            WriteValue(builder, "optionstrict", Options.OptionStrict.ToString())
+
+            Dim isOptimized = If(Options.OptimizationLevel = OptimizationLevel.Release, True, False)
```

Comment:
>isOptimized [](start = 16, length = 11)

check if VB also supports `debug+`

Reply:
Storing optimization as the enum value instead of bool, so we don't need to worry about this anymore afaik